src/gr-unit.c

```diff
@@ -16,152 +16,154 @@
  * along with this program.  If not, see <http://www.gnu.org/licenses/>.
  */
 
-#include "config.h"
```

Comment:
What happened here ? change of line endings ? the whole file seems to have been replaced without change ?